Makefile

```diff
@@ -14,6 +14,8 @@ generate-ci:
 generate-serving-ci:
 	rm -rf openshift openshift-knative
 	go run github.com/openshift-knative/hack/cmd/prowgen --config config/serving.yaml --remote $(REMOTE)
+	go run github.com/openshift-knative/hack/cmd/prowgen --config config/serving-net-istio.yaml --remote $(REMOTE)
+	go run github.com/openshift-knative/hack/cmd/prowgen --config config/serving-net-kourier.yaml --remote $(REMOTE)
```

Comment:
Can we add these 2 to `generate-ci`? or alternatively we could define `generate-ci` to run `generate-eventing-ci` and `generate-serving-ci` to avoid the duplication

Reply:
Sure, I added it as duplication as the `rm -rf openshift openshift-knative` part makes it harder to depend on another target. Alternative is to have more conditionals somehow to make both cases work.